<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DRuGLORD" failures="9" tests="9" name="com.cab.cucumber.runners.RunnerUsingTestNG" time="34.998" errors="0" timestamp="2025-08-19T11:59:43 IST" skipped="0">
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.178">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.164">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.259">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="6.955">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.164">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.170">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.284">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="5.561">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.263">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.cab.cucumber.runners.RunnerUsingTestNG -->
